MetaMask/metamask-mobile
# feat: add MetaMetrics delete on Wallet delete

## **Description**
This creates a MetaMetrics deletion task when a user deletes their wallet.

## **Related issues**

Fixes: [1271](https://github.com/MetaMask/mobile-planning/issues/1271)

## **Manual testing steps**

1. Open Flipper Network inspector
2. Go to Settings > Security
3. Delete Wallet
4. Should see Successful call to MM proxy

## **Screenshots/Recordings**



### **Before**



### **After**


https://github.com/MetaMask/metamask-mobile/assets/141057783/45472d73-29c8-4e00-ad43-74bb774a51c0



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
